README.md

```diff
@@ -89,25 +89,13 @@ Please follow this design to ensure the ssm keys are easy to maintain and naviga
 
 ## Generating VPN client keys
 
```

Comment:
We should also explain the usage of `generate_vpn_ca` and `generate_vpn_server_crt` tasks